lib/Doctrine/ruleset.xml

```diff
@@ -93,6 +93,12 @@
     </rule>
     <!-- Forbid dead code -->
     <rule ref="SlevomatCodingStandard.Classes.UnusedPrivateElements"/>
+    <!-- Forbid suffix "Abstract" for abstract classes -->
+    <rule ref="SlevomatCodingStandard.Classes.SuperfluousAbstractClassNaming"/>
```

Comment:
Should lines 97 and 101 be moved to other PRs? Or should I add them to the standard?

Reply:
I add all of them ‘cause of https://github.com/doctrine/coding-standard/pull/31#issuecomment-368410442

Reply:
Ok let me document that too then.

Reply:
`SuperfluousAbstractClassNaming` and `SuperfluousInterfaceNaming` forbid both suffix and prefix. So you should document this or add `<exclude>` to ignore the prefix check.

Reply:
@greg0ire I guess we want just the suffix here, right?

Reply:
Aaah ok yes, I think that's it, because I saw a lot of `Abstract` prefixes in the code.